### PR TITLE
Disable title and chat messages for Melon/Pumpkin Dicer

### PR DESCRIPTION
**Pull request by Ghost-3**
Removes the annoying flashing title when using Dicer.
Also filters messages about drops in chat. (Disabled by default)

### CODE REVIEW

File: src/main/java/de/hysky/skyblocker/config/SkyblockerConfig.java [hunk 1]
```diff
@@ -1092,4 +1098,9 @@ public String toString() {
 			return I18n.translate("text.autoconfig.skyblocker.option.richPresence.info." + name());
 		}
 	}
+
+	public static class Garden {
+		@SerialEntry
+		public boolean dicerTitlePrevent = true;
+	}
```
Comment (kevinthegreat1) [inline src/main/java/de/hysky/skyblocker/config/SkyblockerConfig.java:1105]: Can you put this right below the spider's den class, please? This makes everything easier to track and gives fewer merge conflicts.

File: src/main/resources/assets/skyblocker/lang/en_us.json [hunk 2]
```diff
@@ -168,6 +168,8 @@
   "text.autoconfig.skyblocker.option.locations.spidersDen.relics": "Hidden Relics Helper",
   "text.autoconfig.skyblocker.option.locations.spidersDen.relics.enableRelicsHelper": "Enable Hidden Relics Helper",
   "text.autoconfig.skyblocker.option.locations.spidersDen.relics.highlightFoundRelics": "Highlight found relics",
+  "text.autoconfig.skyblocker.option.locations.garden": "Garden",
+  "text.autoconfig.skyblocker.option.locations.garden.dicerTitlePrevent": "Enable dicer title prevent",
```
Comment (kevinthegreat1) [inline src/main/resources/assets/skyblocker/lang/en_us.json:172]: Can you capitalize all the words in the string and while you're at it also the found relics string?

**Review by kevinthegreat1 — CHANGES_REQUESTED** [review]
Code looks good, just some small things. Didn't test yet.

**Comment by Ghost-3** [comment]
All fixed.

**Review by kevinthegreat1 — COMMENTED** [review]
Can you confirm that it's still working after the latest changes? I'm not sure how to test it. Then I'll approve it.

**Review by kevinthegreat1 — APPROVED** [review]

**Pull request merged**